Pyright fails to start

### Summary

Pyright fails to start when opening a .py file

Steps to trigger the problem:
1.  Start Zed
2. Open a .py file


Actual Behavior:
Zed reports an LSP error

Expected Behavior:
Pyright loads successfully.


Zed reports an error when it attempts to load Pyright, when I open a .py file.  


Error message:
Language server error: pyright

oneshot canceled
-- stderr--
Unexpected option --stdio.
pyright --help for usage



Relevant Zed log entries:
2025-02-10T09:47:51.595638-06:00 [ERROR] cannot read LSP message headers
2025-02-10T09:47:51.5962048-06:00 [ERROR] Failed to start language server "pyright": oneshot canceled
2025-02-10T09:47:51.5964684-06:00 [ERROR] server stderr: "Unexpected option --stdio.\npyright --help for usage\n"
2025-02-10T09:47:51.5967167-06:00 [ERROR] The pipe is being closed. (os error 232)
2025-02-10T09:47:51.5970044-06:00 [ERROR] server shut down 



### Zed Version and System Specs

Zed: v0.174.0 (Zed Dev 6f7f0f30e280b0d896668b27b8be62b4c212f9c7) 
OS: Windows 10.0.19045
Memory: 16 GiB
Architecture: x86_64
GPU: NVIDIA GeForce GT 1030 || NVIDIA || 572.16

--- a/crates/languages/src/python.rs
+++ b/crates/languages/src/python.rs
@@ -83,7 +83,7 @@ impl LspAdapter for PythonLspAdapter {
         _: Arc<dyn LanguageToolchainStore>,
         _: &AsyncApp,
     ) -> Option<LanguageServerBinary> {
-        if let Some(pyright_bin) = delegate.which(Self::SERVER_NAME.as_ref()).await {
+        if let Some(pyright_bin) = delegate.which("pyright-langserver".as_ref()).await {
             let env = delegate.shell_env().await;
             Some(LanguageServerBinary {
                 path: pyright_bin,

